### PR TITLE
ed25519: enable feature gate in generator

### PR DESCRIPTION
**Pull request by 0x0ece**
Actual tests: https://github.com/firedancer-io/test-vectors/pull/29

### CODE REVIEW

File: generators/ed25519.py [hunk 1]
```diff
@@ -1533,7 +1533,7 @@ def _into_key_data(key_prefix, test_vectors):
     account.address = program_id
     account.owner = program_owner
     instr_ctx.accounts.extend([account])
-    instr_ctx.instr_accounts.extend([invoke_pb.InstrAcct()])
```
Comment (ravyu-jump) [inline generators/ed25519.py:1536]: Why was this removed?

Comment (0x0ece) [reply generators/ed25519.py:1536]: it's no longer needed. an instruction can now have 0 accounts
(the tx must have 1+ account, in this case the program)

**Review by ravyu-jump — APPROVED** [review]
